dev/devicelab/lib/tasks/integration_tests.dart

```diff
@@ -208,7 +208,12 @@ class DriverTest {
         ...extraOptions,
       ];
       await flutter('drive', options: options, environment: environment);
-
+      await flutter('install', options: <String>[
+        '--uninstall-only',
+        '-t', testTarget,
```

Comment:
I think this line is causing a build failure.

Reply:
Yeah, I spotted that last night but haven't had time to fix it yet.